src/components/MovieDescription/MovieDescription.js

```diff
@@ -0,0 +1,20 @@
+import './MovieDescription.css';
+
+const MovieDescription = ({info}) => {
+    return (
+        <div>
+            <div>
+                <h3>{info.tagline}</h3>
+                <p>{info.overview}</p>
+            </div>
+            <div>
+                <h3>Stats</h3>
+                <p>Budget: {info.budget}</p>
+                <p>Revenue: {info.revenue}</p>
+                <p>Runtime: {info.runtime} minutes</p>
+            </div>
```

Comment:
Do these need to be in separate divs?

Reply:
I wanted them to be side by side, I just didnt do the styling yet. open to changing!

Reply:
That makes sense!